### PR TITLE
Update overall

### PR DESCRIPTION
**Pull request by Siphalor**

### CODE REVIEW

File: src/components/footer/footer.html [hunk 1]
```diff
@@ -1,7 +1,7 @@
-<div class="footer">
+<footer class="footer">
```
Comment (flyingP0tat0) [inline src/components/footer/footer.html]: Vue likes you to only use standard HTML tags to prevent confusion when using components later on. I think nothing speaks against using plain `div`s.

File: src/components/news/news.js [hunk 2]
```diff
@@ -1,14 +1,17 @@
 // @flow
 
+var lorem = require('lorem-ipsum')
```
Comment (flyingP0tat0) [inline src/components/news/news.js]: For a webpack project, please use the
```javascript
import name from 'module'
```
syntax instead of a `require` statement.

File: src/components/vplan/vplan.scss [hunk 3]
```diff
@@ -1,6 +0,0 @@
-.card {
```
Comment (flyingP0tat0) [inline src/components/vplan/vplan.scss]: Why did you delete the vplan styles?

File: src/components/footer/footer.html (hunk 1 above)
Comment (Siphalor) [reply src/components/footer/footer.html]: So why are you using `nav` in the menu component.

Shouldn't we use **either** old HTML **or** HTML5?

File: src/components/news/news.js (hunk 2 above)
Comment (Siphalor) [reply src/components/news/news.js]: Ok, I just copied it from the official documentation.

File: src/components/vplan/vplan.scss (hunk 3 above)
Comment (Siphalor) [reply src/components/vplan/vplan.scss]: I changed the styling, so it's styled completely via vanilla bootstrap classes. I think the styling was a bit broken before.

Just create a new branch and checkout whether you like the style changes or not.

File: src/components/footer/footer.html (hunk 1 above)
Comment (flyingP0tat0) [reply src/components/footer/footer.html]: Because of vue's recommondation, I will also change the `nav` tag.

**Comment by flyingP0tat0** [comment]
Thanks you! I reverted the change about vplan. You are right about the table looking somewhat odd. But the styling is the only way I got it to work on smaller screens yet. Otherwise it would just display the table without any background and therefor laying directly on the background image.
